tests/network/migration/test_migration.py

```diff
@@ -398,23 +398,23 @@ def test_connectivity_after_migration_and_restart(
     )
 
 
-@pytest.mark.polarion("CNV-2061")
 @pytest.mark.s390x
+@pytest.mark.usefixtures("http_service", "migrated_vmb_and_wait_for_success")
+@pytest.mark.parametrize(
+    "ip_family",
+    [
+        pytest.param("ipv4", marks=[pytest.mark.ipv4, pytest.mark.polarion("CNV-12508")], id="ipv4"),
+        pytest.param("ipv6", marks=[pytest.mark.ipv6, pytest.mark.polarion("CNV-12509")], id="ipv6"),
```

Comment:
Please change the IDs, or remove them completely. The ID should be unique and reflect the specific test that it represents. Having the IP family as a test ID is confusing.